dependencies/libgit2/meson.build

```diff
@@ -0,0 +1,11 @@
+# libgit2 Dependency File
+#============================================================================
+
+
+# FIXME !! lowdown is seemingly a required dependency for building, but 
+# it isn't listed anywhere on documentation that i could find. dependency
+# object needs to be added to meson.
+
+# Look for lowdown, a required dependency
```

Comment:
These comments does not seem relevant for libgit2, copy pasta?

Reply:
looks to be :)